monero-outreach-docs/translations/pt-br/storyboard/tool_for_renewal_pt_br.md

```diff
@@ -0,0 +1,77 @@
+
+Contudo, sendo novo, Bitcoin teve falhas técnicas, incluindo pouca privacidade. E então outro personagem ou grupo pseudônimo atendido pelo nome de Nicolas van Saberhagen desenvolveu o protocolo CryptoNote. Desenvolvedores tomaram conhecimento e assim nasceu o Monero. O Monero resolveu vários problemas do Bitcoin, como, por exemplo, este último ser facilmente vigiado por companhias e governos. A Chainalysis, uma empresa de análise de blockchain, declarou recentemente em um webinar público, [chainalysis.com/.../Advanced-Obfuscation-Techniques-Public-Webinar.mp4](https://go.chainalysis.com/rs/503-FAP-074/images/Advanced-Obfuscation-Techniques-Public-Webinar.mp4), “Monero é muito mais focada em privacidade do que seus similares… Tudo no Monero é ofuscado, você não pode distinguir nada."
+
+Monero é totalmente código aberto e tem uma das mais ativas e disseminadas comunidades de todas as criptomoedas. A comunidade tem ganhado reputação para um ethos que suporta autodeterminação, soberania, e a proteção dos direitos humanos. Se você está procurando por um sistema que pode tirar poder de governos corruptos e banqueiros centrais - e ainda melhorar a forma como as pessoas interagem e transacionam umas com as outras – nós convidamos você a usar Monero.
+
+Quando uma transação é realizada usando Monero, o remetente, o receptor, e a quantia são todos ocultados usando criptografia, prevenindo assim que governos, empresas, e outros, vejam e gravem suas ações. Com Monero, compradores e vendedores individuais obtém a proteção à privacidade que se espera para atividades pessoais em sua própria casa. Matemática e software de código aberto protegem você, sem envolver bancos, sistema legal, ou a polícia. 
+
+Seja se você precisa mover seu dinheiro para uma poupança privada e digital, ou seja pelo seu próprio desejo de não participar do sistema financeiro atual, junte-se a nós. Juntos, nós podemos usar novas tecnologias para limitar a intromissão de banqueiros e governos corruptos, e participar de um sistema financeiro que capacita e dá poder às pessoas. Estamos todos juntos nisso. Vamos usar estas ferramentas para sermos a mudança que desejamos ver no mundo, por nós e pelas futuras gerações.
+
+Nós não esperamos que você aceite nossas palavras por nada disso e encorajamos você a fazer sua própria pesquisa e verificar com fontes. Nós compartilharemos alguns pontos iniciais aqui. Concluindo sua pesquisa, por favor volte para o guia rápido sobre como iniciar a usar o Monero (veja abaixo).
+
+### _Leitura Complementar:_
+
+- O que é Monero? (Website Oficial) [web.getmonero.org/get-started/what-is-monero](https://web.getmonero.org/get-started/what-is-monero/)
```

Comment:
Hey @nelsonrenan 

For the getmonero links you can use the translated pt-br version from the site. Take a look https://www.getmonero.org/pt-br/get-started/what-is-monero/index.html.

That way the user will be redirected to the language version. That gives a better user experience. Not all of the links are translated but if we find any that has the translation ready we could use it. 

Have that in mind for future translations.